migrations/versions/0376_add_org_names.py

```diff
@@ -0,0 +1,29 @@
+"""
+
+Revision ID: 0376_add_org_names
+Revises: 0375_fix_service_name
+Create Date: 2022-09-23 20:04:00.766980
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0376_add_org_names'
+down_revision = '0375_fix_service_name'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.get_bind()
+
+    op.execute("INSERT INTO organisation_types VALUES ('state','f','250000'),('federal','f','250000');")
```

Comment:
Should we have a line to remove unused org types from the db?

Reply:
Yeah, removing them didn't seem to cascade cleanly, so I left them in to avoid breaking the demo. But we shouldn't have that issue when we set up a fresh instance, so I'll add it and we can hold off on merging until after we get our new space.